toolchains/cc/cc.nix

```diff
@@ -37,20 +37,19 @@ let
           paths =
             if pkgs.stdenv.isDarwin then
               {
-                cc = (pkgs.overrideCC pkgs.stdenv darwinCC).cc;
-                binutils = pkgs.darwin.binutils;
+                inherit (darwinCC) cc bintools;
               }
             else
               {
                 cc = pkgs.stdenv.cc;
-                binutils = pkgs.binutils;
+                bintools = pkgs.binutils;
```

Comment:
should we just use `pkgs.stdenv.cc.bintools` here for consistency too?

Reply:
They seem to be the same by default:
```
# MacOS
nix-repl> darwin.binutils
_derivation /nix/store/rvjqxkb0df4nlv4lvqljshf7ac3y54c8-cctools-binutils-darwin-wrapper-949.0.1.drv_

nix-repl> stdenv.cc.bintools
_derivation /nix/store/rvjqxkb0df4nlv4lvqljshf7ac3y54c8-cctools-binutils-darwin-wrapper-949.0.1.drv_
```
```
# Linux
nix-repl> stdenv.cc.bintools
«derivation /nix/store/fwywivmkyn0mg7qbsy161zzqfv9kq415-binutils-wrapper-2.35.2.drv»

nix-repl> binutils
«derivation /nix/store/fwywivmkyn0mg7qbsy161zzqfv9kq415-binutils-wrapper-2.35.2.drv»
```
I was thinking that if somebody alters the `stdenv` of the packages set we get passed in, it would be counter-intuitive if we use the `cc` from `stdenv` (which was overridden) but insist on using the plain `binutils` package.

Reply:
Thanks for checking, yes, in that case being consistent seems like the better approach.